src/Foundatio/Messaging/SharedMessageBusOptions.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Foundatio.Serializer;
+using Foundatio.Utility;
+using Microsoft.Extensions.Logging;
+
+namespace Foundatio.Messaging {
+    public class SharedMessageBusOptions : SharedOptions {
```

Comment:
Shall we should keep the original class name `MessageBusOptionsBase` to avoid breaking changes for the implementation libraries.

Reply:
I'm not to worried, this was made in 6.0 (a month or two ago) and I doubt anyone besides us has inherited from them.

Reply:
Yeah, 6.0 is a breaking change. I think Shared is a better word and it didn't need to be an abstract class and that was actually causing issues with some of the fluent builder stuff.